pex/pex.py

```diff
@@ -155,9 +155,14 @@ def minimum_sys_modules(cls, site_libs, modules=None):
 
   @classmethod
   def minimum_sys_path(cls, site_libs, inherit_path):
+    ver = sys.version[:3]
     scrub_paths = OrderedSet()
     site_distributions = OrderedSet()
     user_site_distributions = OrderedSet()
+    undesirable_paths = OrderedSet([
+      '/System/Library/Frameworks/Python.framework/Versions/%s/Extras/lib/python' % ver,
```

Comment:
@wickman it seems 'EXTRASPATH' isn't defined in Sierra I'll dig around some more.

as seen in https://github.com/pantsbuild/pex/blob/master/pex/pex.py#L94

```sh
jvenus-signal-mbp13:clowncar jvenus$ /usr/bin/python
Python 2.7.10 (default, Jul 30 2016, 18:31:42)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from distutils import sysconfig
>>> makefile = sysconfig.parse_makefile(sysconfig.get_makefile_filename())
>>> for l in makefile.keys():
...   if 'EXT' in l:
...     print l
...
EXTRAPLATDIR
EXTRATESTOPTS
EXTRA_CFLAGS
EXTRAMACHDEPPATH
```